docs/src/syntax-highlighting/rehype-inline-code.mjs

```diff
@@ -1,5 +1,6 @@
 import { visitParents } from 'unist-util-visit-parents';
 
+/** Adds basic syntax highlighting to inline `<code>` elements */
 export function rehypeInlineCode() {
```

Comment:
Unrelated, but I'm curious why you're doing this rather than using `rehype-pretty-code`'s inline highlighting feature? Rather than only a single color for the inline code, it accurately highlights it (punctuation differences, etc.) like it were a regular code block.

Reply:
Gave it another go, it's pretty good! Thanks for pointing in out

Initially I couldn't figure out why it wasn't working and didn't have the patience to debug; was just missing the default language option